Modules/Relationships/src/Relationships.Application/Identities/IdentityDeleter.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Backbone.BuildingBlocks.Application.Identities;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Relationships.Application.Relationships.Commands.DeleteRelationshipsByIdentity;
+using Backbone.Modules.Relationships.Application.Relationships.Commands.DeleteRelationshipTemplatesByIdentity;
+using MediatR;
+
+namespace Backbone.Modules.Relationships.Application.Identities;
+public class IdentityDeleter(IMediator mediator) : IIdentityDeleter
+{
+    private readonly IMediator _mediator = mediator;
+
+    public async Task Delete(IdentityAddress identityAddress)
+    {
+        await _mediator.Send(new DeleteRelationshipsByIdentityCommand(identityAddress));
```

Comment:
Should we validate the outcome of this command before proceeding?

Reply:
I don't see a reason to. Going down the call chain, we can see we're simply deleting a few entities via a repo. In the unlikely event that that fails, what should we do?

Reply:
Well, at some point we have to document the error in the Audit Log. But I think this should be done by the caller of this method.